docs/src/developer/gpu_computations.md

```diff
@@ -0,0 +1,58 @@
+# GPU computations
+
+GPU computations in DFTK is very much a work in progress. The goal is to build on
+Julia's multiple dispatch to have the same code base for CPU and GPU. We hope we
+will be able to reach decent performances without having to write any kernels at all,
+only by relying on the high level functionalities implemented in the GPU packages.
+
+To go even further with this idea of unified code, we would also like to be able to
+support any type of GPU architecture: unlike other packages, we do not want to
```

Comment:
that's not very nice :-P I think everybody wants that

Reply:
Haha, I didn't want to be mean. I just wanted to point out that we were being quite ambitious as support for multiple GPU architectures doesn't seem to be the standard for the moment.

Reply:
> doesn't seem to be the standard for the moment.

Well. Everyone wants to, but some people struggle to make it happen. But the major codes (e.g. VASP, SIRIUS-QE) have it now for CUDA, AMD and Intel.